Source/SwiftLintFramework/Helpers/XCTestHelpers.swift

```diff
@@ -0,0 +1,24 @@
+import SourceKittenFramework
+
+private let testFunctionNames: Set = [
+    "setUp()",
+    "setUpWithError()",
+    "tearDown()",
+    "tearDownWithError()"
+]
+
+private let testVariableNames: Set = [
+    "allTests"
+]
+
+enum XCTestHelpers {
+    static func isXCTestMember(kind: SwiftDeclarationKind, name: String) -> Bool {
+        if SwiftDeclarationKind.functionKinds.contains(kind) {
+            return name.hasPrefix("test") || testFunctionNames.contains(name)
```

Comment:
Some of these are only test members if they're static or instance functions. For example, the following examples would be incorrectly counted as test members:

```swift
final class MyTests: XCTestCase {
  static func testHelper() -> TestHelper { ... }
}
```